### DISCUSSION TITLE
Language resource file for routes with variables

### DISCUSSION THREAD

**Discussion by akiarostami** [sveltekit-i18n/lib#108] [Q&A] [answered]
Hello,

One of the great features of this library is being able to load a language resource specific to a route in the loaders section of config. However, this seems to only work for static paths (e.g. /blog). I can't find a way to pass a variable part (e.g. /bog/[title], where [title] section changes for each blog entry). Is it possible to do that with this library?

Thank you.

**Comment by jarda-svoboda** [comment, marked as answer]
HI @akiarostami!
If i understand you corectly you just need to pass a RegExp instead of static (string) path into your loader's `paths`.

**Reply by akiarostami** [reply]
Thank you!

[a comment by an automated account is not shown]
